MedicalDocManagment.WebUI/app/admin-http.facade.ts

```diff
@@ -0,0 +1,44 @@
+﻿import { Injectable } from '@angular/core';
```

Comment:
Please change file name on `admin.service.ts` and if you want rename your `http.facade.ts` file on `app.service.ts`.